composer.json

```diff
@@ -44,6 +45,7 @@
             "dev-master": "2.0.x-dev"
         }
     },
+    "minimum-stability": "dev",
```

Comment:
please keep using stable deps for most builds (which allows to benefit from the download cache). A single job should be running with minimum-stability=dev (see Assetic for an example)